api/envoy/admin/v3/config_dump.proto

```diff
@@ -49,13 +49,18 @@ message UpdateFailureState {
       "envoy.admin.v2alpha.UpdateFailureState";
 
   // What the component configuration would have been if the update had succeeded.
+  // This field may not be present on certain implementation of xDS client or
+  // server due to concerns of overhead.
```

Comment:
I don't think this is going to be an issue on the server.  The server will already store the most recent version, because that's what it's expecting to serve.

In contrast, the server may not want to populate the version currently being used by the client if it has NACKed the most recent update, because that would require the server to hold on to an older version of the resource.

Reply:
Comment updated to detail description of the expectation of xDS client and xDS server.

Reply:
The comment about the server isn't relevant to this field, though.  If the client NACKs, then this field is always the most recent version.

I think this comment should only mention the client.  The comment about the server is probably more relevant in the fields that contain the version that the client is currently using.

Reply:
Updated the comment to only mention the xDS client.